src/bridge/views/IssueChild.js

```diff
@@ -250,6 +224,11 @@ class IssueChild extends React.Component {
             value={ state.desiredPoint }
             onChange={ this.handlePointInput }>
             <InnerLabel>{ 'Point to Issue' }</InnerLabel>
+            <div
+              style={{marginTop: '8.8rem', marginLeft: '4.6rem', fontSize: '5rem', color: '#757575', whiteSpace: 'pre-wrap'}}
```

Comment:
I have no idea the classes you're using work, except that tl-0 seems to mean top and left 0! Anyways it's probably terrible to have custom css here but it works.

Reply:
This seems to be doing some very careful positioning and sizing of the text to make it overlap just right. I'm personally fine with leaving this as is, save for maybe splitting the style out over multiple lines.